src/yunohost/tools.py

```diff
@@ -565,3 +565,25 @@ def tools_diagnosis(auth, private=False):
         diagnosis['private']['domains'] = domain_list(auth)['domains']
 
     return diagnosis
+
+def tools_public_key():
+    key_path = '/root/.ssh/id_rsa_ynh'
```

Comment:
I'd vote to have the key stored in something like `/etc/yunohost/keys/` (or whichever folder name sounds right and reflects the idea of 'federated identity').

There were some IRL discussions about how to feed/choose the relevant SSH key with borg backup, and I think the answer is in [here](http://borgbackup.readthedocs.io/en/stable/usage.html) (c.f. `BORG_RSH`)